include/fmt/color.h

```diff
@@ -191,25 +191,49 @@ enum class color : uint32_t {
   yellow_green            = 0x9ACD32  // rgb(154,205,50)
 };  // enum class color
 
+enum class terminal_color : uint8_t {
+  black = 30,
+  red,
+  green,
+  yellow,
+  blue,
+  magenta,
+  cyan,
+  white,
+  bright_black = 90,
+  bright_red,
+  bright_green,
+  bright_yellow,
+  bright_blue,
+  bright_magenta,
+  bright_cyan,
+  bright_white
+};  // enum class terminal_color
+
 enum class emphasis : uint8_t {
   bold = 1,
   italic = 1 << 1,
   underline = 1 << 2,
   strikethrough = 1 << 3
 };  // enum class emphasis
 
-// rgb is a struct for red, green and blue colors.
+// rgb is a struct for red, green and blue colors plus an extra byte to store
+// a terminal color.
```

Comment:
rgb and terminal colors are mutually exclusive so I think it's better to create a separate structure that represents a discriminated union of those.